blueoil/blueoil_init.py

```diff
@@ -272,6 +272,23 @@ def ask_questions():
     }
     training_epochs = prompt(training_epochs_question)
 
+    initial_learning_rate_value_question = {
+        'type': 'input',
+        'name': 'value',
+        'message': 'initial learning rate:',
+        'default': '0.001'
+    }
+    initial_learning_rate_value = prompt(initial_learning_rate_value_question)
+
+    training_learning_rate_question = {
+        'type': 'rawlist',
+        'name': 'value',
+        'message': 'choose learning rate setting(tune1 / tune2 / tune3 / fixed):',
+        'choices': ['tune1 -> "2 times decay"', 'tune2 -> "3 times decay"', 'tune3 -> "warm-up and 3 times decay"', 'fixed'],
```

Comment:
In the CLI, the description looks good. 
<img width="444" alt="screen shot 2018-11-14 at 21 34 50" src="https://user-images.githubusercontent.com/138891/48482900-43abd900-e855-11e8-869f-81452634ba2d.png">

But in the generated yml, `tune3 -> "warm-up and 3 times decay"` looks long, it is difficult to change.
```
$ cat config/xxxx.yml
...
...
trainer:
  batch_size: 10
  epochs: 100
  training_learning_rate: tune3 -> "warm-up and 3 times decay"
  initial_learning_rate: 0.1

....
....
```

We believe that easy change of generated yml by blueoil user will increase usability.
From that point of view, In the generated yml, User only seems to need key like `tune3`, don't need description like `tune3 -> "warm-up and 3 times decay"`.

The below pseudo code will solve this. 
```
choices_key_map = {
  'tune1 -> "2 times decay"'': 'tune1',
  'tune2-> "3 times decay"'': 'tune2',
  'tune3 -> "warm-up and 3 times decay"': 'tune3',
  'fixed': 'fixed',
}
tmp_learning_rate_setting = prompt(training_learning_rate_question)
training_learning_rate_setting = choices_key_map[tmp_learning_rate_setting]
```